Detectors/ITSMFT/ITS/tracking/src/TrackerTraits.cxx

```diff
@@ -394,8 +395,13 @@ void TrackerTraits::findCellsNeighbours(const int iteration)
 
     int layerCellsNum{static_cast<int>(mTimeFrame->getCells()[iLayer].size())};
     const int nextLayerCellsNum{static_cast<int>(mTimeFrame->getCells()[iLayer + 1].size())};
-    mTimeFrame->getCellsNeighbours()[iLayer].resize(nextLayerCellsNum);
 
+    mTimeFrame->getCellsNeighboursLUT()[iLayer].clear();
+    mTimeFrame->getCellsNeighboursLUT()[iLayer].resize(nextLayerCellsNum, 0);
+    std::vector<std::pair<int, int>> cellsNeighbours;
+    cellsNeighbours.reserve(nextLayerCellsNum);
+
+    std::vector<std::vector<int>> easyWay(nextLayerCellsNum);
```

Comment:
@mpuccio in my profiles I see this line as responsible of 80% of the malloc / free calls for its-tracker. Don't we have any other way of doing the same, without the churn?

Reply:
Sure, this is actually a debug vector that I forgot to remove. I will remove it later in the morning

Reply:
@mpuccio Apparently it's still there. Is there any issue with removing it?

Reply:
I removed the `easyWay` vector; do you want to remove cellsNeighbours? This is needed; I can add a reserve if you see that this slows down the processing (I honestly did not see that).

Reply:
No, i guess I simply mis remembered which vector. Sorry for the noise. I will profile again an let you know if anything else is needed.